src/App.js

```diff
@@ -1,36 +1,36 @@
-import { useEffect, useState } from "react";
```

Comment:
Atenção na ordem da importação 

https://dev.to/coderamos/hierarquia-de-imports-56j6